src/components/FormPage.jsx

```diff
@@ -0,0 +1,13 @@
+"use client";
+import React from "react";
+
+const FormPage = ({ title, children, restrictions }) => {
```

Comment:
- remove restrictions since its not being used
- call the component Form.jsx
- Remove the use client, this doesn't need it, its children need it